styles.js

```diff
@@ -33,23 +46,69 @@ export const StyledList = styled.ul`
   justify-content: space-between;
   list-style: none;
   gap: 1rem;
-  border: 1px solid rebeccapurple;
 `;
 
 export const StyledItem = styled.li`
+  background-color: #2d3250;
+  color: #ffffff;
   border: 1px solid;
   margin: 10px 5px 10px 5px;
-  position: relative;
+  display: flex;
   padding: 7px;
+  position: relative;
 `;
 
 export const StyledHint = styled.p`
-  font-size: 9px;
+  font-size: 13px;
   text-align: center;
+  color: #ffb228;
+`;
+
+export const StyledDeleteDiv = styled.div`
+  margin: 5%;
 `;
 
 export const StyledDeleteButton = styled.button`
   display: flex;
```

Comment:
I think you don't need the display: flex and justify-content because it would just apply to children elements of the button but there are none. But I am not 100% sure, maybe you can try it out.